rules/duo_rules/duo_admin_lockout.py

```diff
@@ -0,0 +1,13 @@
+def rule(event):
+    # Return True to match the log event and trigger an alert.
+    return event.get("action", "") == "admin_lockout"
+
+
+def title(event):
+    # If no 'dedup' function is defined, the return value
+    # of this method will act as deduplication string.
+    return (
+        f"Duo Admin [{event.get('username', '<NO_USER_FOUND>')}] is "
+        f"locked out. Reason: "
+        f"[{event.get('description', {}).get('message','<NO_MESSAGE_FOUND>')}]."
```

Comment:
I expect that this is aways returning `<NO_MESSAGE_FOUND>` from the test suite. `event.description` is a string which needs to be `json.loads()`'ed to understand that there is a `message` attribute in there.